{{cookiecutter.project_slug}}/{{cookiecutter.project_slug}}/wsgi.py

```diff
@@ -7,11 +7,5 @@
 https://docs.djangoproject.com/en/{{ cookiecutter.django_version }}/howto/deployment/wsgi/
 """
 
-import sys
-
-# Python 2 threading problems
-if sys.version_info < (3,):
```

Comment:
All of our new projects are Python 3, so lets standardise on it. So why this for python2 ?